@xen-orchestra/web-core/lib/components/ui/toaster/UiToaster.vue

```diff
@@ -0,0 +1,103 @@
+<!-- v3 -->
+<template>
+  <div :class="toVariants({ accent })" class="ui-toaster">
+    <div class="label">
+      <VtsIcon class="information-icon" :accent :icon="faCircle" :overlay-icon="icon" />
+      <div class="content">
+        <div class="typo h5-semi-bold">
+          <slot />
+        </div>
+        <div v-if="slots.description" class="description typo p2-regular">
+          <slot name="description" />
+        </div>
+      </div>
+      <UiButtonIcon class="close-icon" :icon="faXmark" :accent size="medium" />
+    </div>
+    <div v-if="slots.actions" class="actions">
+      <slot name="actions" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import { toVariants } from '@core/utils/to-variants.util'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { faXmark, faCheck, faCircle, faInfo, faExclamation } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+type ToasterAccent = 'info' | 'success' | 'warning' | 'danger'
+
+const props = defineProps<{
+  accent: ToasterAccent
+}>()
+
+const slots = defineSlots<{
+  default(): any
+  description?(): any
+  actions?(): any
+}>()
+
+const states: Record<ToasterAccent, { icon: IconDefinition; color: ToasterAccent }> = {
+  info: { icon: faInfo, color: 'info' },
+  success: { icon: faCheck, color: 'success' },
+  warning: { icon: faExclamation, color: 'warning' },
+  danger: { icon: faXmark, color: 'danger' },
+}
+
+const icon = computed(() => states[props.accent].icon)
```

Comment:
Maybe it was for a test, but I think the `color` key in the object is not useful here, and this code can be simplified:
```suggestion
const states: Record<ToasterAccent, IconDefinition> = {
  info: faInfo,
  success: faCheck,
  warning: faExclamation,
  danger: faXmark,
}

const icon = computed(() => states[props.accent])
```